cmd/helm/reset_test.go

```diff
@@ -55,7 +55,7 @@ func TestResetCmd(t *testing.T) {
 	if len(actions) != 2 {
 		t.Errorf("Expected 2 actions, got %d", len(actions))
 	}
-	expected := "Tiller (the helm server side component) has been uninstalled from your Kubernetes Cluster."
+	expected := "Tiller (the helm server-side component) has been uninstalled from your Kubernetes Cluster."
```

Comment:
- [ ] While we're still pursuing these corrections, why is "helm" written in lowercase here?  
  Did we miss that in the _reset.go_ file?